### PR TITLE
Add r-admm

### PR DESCRIPTION
**Pull request by TanguyGen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TanguyGen** [comment]
@conda-forge/help-r

File: recipes/r-admm/meta.yaml
Comment (mfansler) [inline recipes/r-admm/meta.yaml:24]: The preferred approach to this is done in this PR: https://github.com/conda/conda-build/pull/4786

```suggestion
    - '*/R.dll'        # [win]
    - '*/Rlapack.dll'  # [win]
    - '*/Rblas.dll'    # [win]
```

File: recipes/r-admm/meta.yaml
Comment (mfansler) [inline recipes/r-admm/meta.yaml:40]: These shouldn't be here:

```suggestion
```

File: recipes/r-admm/meta.yaml
Comment (mfansler) [inline recipes/r-admm/meta.yaml:63]: The only thing needed here should be:

```suggestion
    - {{ native }}libwinpthread-git  # [win]
```

The other stuff will automatically get injected as run exports.

File: recipes/r-admm/meta.yaml
Comment (mfansler) [inline recipes/r-admm/meta.yaml:72]: The DESCRIPTION file shows license is `GPL (>= 3)`, so:

```suggestion
  license: GPL-3.0-or-later
```

File: recipes/r-admm/meta.yaml
Comment (mfansler) [inline recipes/r-admm/meta.yaml:50]: Instead, keep BLAS implementation open, and put OpenMP dependency here:

```suggestion
    - libblas
    - libgomp          # [linux]
    - llvm-openmp      # [osx]
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Thanks for submitting this. There are a few things to tidy up before merging.

**Review by mfansler — COMMENTED** [review]
Everything here looks good, but there is something strange in the logs for the Windows build (also builds the recipe `tyro`). I'm going to rerun the checks and see if it resolves itself.

**Comment by TanguyGen** [comment]
Thank you really much for reviewing this package !

**Comment by mfansler** [comment]
Sorry for the technical difficulties - unsure why the CI is being buggy. Anyway, this round seems to be working, so I'll merge once I confirm no a clean run. 🤞

**Comment by TanguyGen** [comment]
It's kind of strange I didn't have this issue before but I looked into it earlier during the precedent built when it was still running and I saw no trace of Tyro until the very end of the build 
Thanks again !!

**Review by mfansler — APPROVED** [review]

**Pull request merged**
